### DISCUSSION TITLE
Regular expression on components

### DISCUSSION THREAD

**Discussion by mary-josephine-1** [buefy/buefy#3931] [Q&A] [answered]
Please guide on how to use Regular expressions on components. 

### Overview of the problem

**OS/Browser**: Ubuntu
```
    "nuxt": "^2.15.8",
    "nuxt-buefy": "^0.4.26",
    "vue": "^2.7.10",
```

### Description
Is there a solution available that can be used to apply **Regular Expressions** like **rules** in **Vuetify**?
Link -  https://v2.vuetifyjs.com/en/api/v-text-field/#props-rules

### Expected behavior
Props for applying regular expressions similar to rules in Vuetify.

### Actual behavior
Currently, no options are found

**Comment by kikuomax** [comment]
@mary-josephine-1 I think Q&A is more appropriate.

**Reply by wesdevpro** [reply]
Is this question even relevant to Buefy? @kikuomax

**Reply by kikuomax** [reply, marked as answer]
@wesdevpro IDK.
@mary-josephine-1 We need more information about your question to answer; e.g., which Buefy component are you asking about?

**Reply by wesdevpro** [reply]
I am pretty sure Vuetify and Buefy are two separate Components frameworks

**Reply by mary-josephine-1** [reply]
@kikuomax 
Vuetify rules is a prop that helps us write our own Regular Expression logic for field-level validations. I couldn't find a solution for this in Buefy.
Regular Expression can be useful for most elements under Form controls.

**Reply by mary-josephine-1** [reply]
@wesdevpro 
I agree. As mentioned above, regular expression is a great option. 
Either Vuetify or Buefy, it's built for user experience. right?

**Reply by wesdevpro** [reply]
> 
> 
> @wesdevpro 
> 
> I agree. As mentioned above, regular expression is a great option. 
> 
> Either Vuetify or Buefy, it's built for user experience. right?

I must admit that I completely read your post wrong here. My apologies. That makes much more sense now

**Reply by mary-josephine-1** [reply]
@wesdevpro 
Thanks for understanding.
